src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final String[] REPORT_CONTENT = {"supply", "buy", "result"};
+    private static final String COMMA_SEPARATOR = ",";
+    private static final String REGEX = "\\r?\\n|\\r";
+    private static final int SUPPLY_INDEX = 0;
+    private static final int BUY_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(composeReport(readFromFile(fromFileName)), toFileName);
+    }
+
+    public void writeToFile(String data, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data", e);
+        }
+    }
+
+    public int[] readFromFile(String fromFileName) {
```

Comment:
Method `readFromFile `must **read From File** only!
Make it similar to writeToFile, return `String `(or `List<String>`) and move logic to composeReport